upload/custom/templates/Bootstrap Material/footer.tpl

```diff
@@ -25,7 +25,11 @@
 	    {*Social media*}
 		{if !empty($SOCIAL_MEDIA_ICONS)}
 		  {foreach from=$SOCIAL_MEDIA_ICONS item=icon}
-		    <a href="{$icon.link}" target="_blank"><i id="social-{$icon.short}" class="fa fa-{$icon.long}-square fa-3x social"></i></a>
+		    {if $icon.link == '/contact/'}
```

Comment:
Thanks for the PR! Could this line be changed to
```
{if $icon.link|strstr:"contact"}
```
as some installations might be located in a subfolder, so /contact/ wouldn't always apply

Reply:
Done